### ISSUE TITLE
Addon doesn't start

### ISSUE THREAD

**Issue by DavyRoswinkel** [MaxWinterstein/toogoodtogo-ha-mqtt-bridge#48]
Since the OS update (8.2), the addon won't start anymore.
```
Add-on: TooGoodToGo Home Assistant MQTT Bridge
 Publish TooGoodToGo stock as MQTT messages
-----------------------------------------------------------
 Add-on version: 2.1.0
 You are running the latest version of this add-on.
 System: Home Assistant OS 8.2  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2022.6.6
 Home Assistant Supervisor: 2022.05.3

```
It's showing several errors, see logfile:

```
[09:06:07] INFO: Strating Bridge...
2022-06-15 09:06:18 [INFO] Connecting mqtt
2022-06-15 09:06:18 [INFO] Starting loop
2022-06-15 09:06:19 [DEBUG] Loaded tokens form tokenfile. Logging in with tokens.
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/app/toogoodtogo_ha_mqtt_bridge/main.py", line 206, in loop
    tgtg_client.login()
  File "/usr/local/lib/python3.8/site-packages/tgtg/__init__.py", line 121, in login
    self._refresh_token()
  File "/usr/local/lib/python3.8/site-packages/tgtg/__init__.py", line 111, in _refresh_token
    raise TgtgAPIError(response.status_code, response.content)
tgtg.exceptions.TgtgAPIError: (403, b'<html><head><title>apptoogoodtogo.com</title><style>#cmsg{animation: A 1.5s;}@keyframes A{0%{opacity:0;}99%{opacity:0;}100%{opacity:1;}}</style></head><body style="margin:0"><p id="cmsg">Please enable JS and disable any ad blocker</p><script>var dd={\'cid\':\'AHrlqAAAAAMAIyk6V6O1U04AU_cwAw==\',\'hsh\':\'1D42C2CA6131C526E09F294FE96F94\',\'t\':\'fe\',\'r\':\'b\',\'s\':35558,\'e\':\'004c485726f6c8bcd2fe16cf789e5171cbcd92f83fbb2e8f97e50ec9c98a109d\',\'host\':\'geo.captcha-delivery.com\'}</script><script src="https://ct.captcha-delivery.com/c.js"></script></body></html>\n')
```

**Comment by Dielee** [comment]
This has nothing to do with the update.
Is fixed in #49.

**Comment by DavyRoswinkel** [comment]
> This has nothing to do with the update. Is fixed in #49.

I will wait for the update of the HA addon then. Thank you!
